Gemfile.lock

```diff
@@ -8,63 +8,71 @@ GIT
 
```

Comment:
you have `upgraded` a lot of gems. Do you really need it? if yes, put it to diff commit, if no just return old one. 

Upgrading can break something, so, let's decide to upgrade if we have reason for that.